nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/AccessResource.java

```diff
@@ -412,7 +412,7 @@ private LogoutRequest completeLogoutRequest(final HttpServletResponse httpServle
     }
 
     private String getNiFiLogoutCompleteUri() {
-        return getNiFiUri() + "logout-complete";
+        return getNiFiUri() + "#/logout-complete";
```

Comment:
This will be a breaking change for existing deployments given that identity providers require registration of logout URLs. What do you think about considering a redirect from `/nifi/logout-complete` in the Jetty Server configuration?

Reply:
Good point. However, we cannot use that path because we use hash based routing. The hash is inserted and all routes begin after it. This allows the front end to handle routing independent of the path the application is deployed to. Switching to path based routing is a possibility but is a larger effort. Path based routing typically requires knowing the base path at build time. There may be options for setting it at run time but this would need to be investigated more thoroughly. 

I was actually just evaluating this in the context of OIDC and there was an issue with hash being encoded. So we'll need to consider the options here.

Reply:
Must have misread your comment initially but we may be able to manually add a Jetty config that just redirects `/nifi/logout-complete` to `/nifi/#/logout-complete`. This is a good idea and would address the other issue I noted above. Will try it out. 👍

Reply:
Thanks for making the adjustments in `JettyServer` to add the rewrite handling @mcgilman, the latest version looks good!